tests/DpBadge.spec.js

```diff
@@ -0,0 +1,44 @@
+import { shallowMount } from '@vue/test-utils'
+import DpBadge from '~/components/DpBadge/DpBadge.vue'
+
+describe('DpBadge', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = shallowMount(DpBadge, {
+      props: {
+        color: 'default',
+        size: 'medium',
+        text: 'Test Badge'
+      }
+    })
+  })
+
+  it('renders the correct text', () => {
+    expect(wrapper.text()).toBe('Test Badge')
+  })
+
+  it('applies the correct color class', () => {
+    expect(wrapper.classes()).toContain('color-text-default')
+    expect(wrapper.classes()).toContain('bg-color-light')
+  })
+
+  it('applies the correct size class', () => {
```

Comment:
I'd do the same as described above for the color prop, and then we can remove the "renders with different size" test below

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/937/commits/8b31152b6f065921247b5950e1d63fac50b74af1